### ISSUE TITLE
Can't connect to MySQL server (initial communication packet error)

### ISSUE THREAD

**Issue by ekstro** [LaraDock/laradock#252]
I'm using LaraDock on Docker/OSX (1.12.0-a). Nginx works great but I have some problems with connecting to MySQL from local machine (eg. from Navicat). MySQL works fine, but it's not available from the local machine ("initial communication packet" error).

What's interesting when I connect to other WiFi (other router) then MySQL on local machine works correctly.

Also I have no problems with "mysql/mysql-server:latest" (change "FROM mysql:latest" in "mysql/Dockerfile"). When I use this package everything works ok.

Can you help me?

**Comment by Mahmoudz** [comment]
@ekstro have you been able to solve this!? 

I was trying to test all the Containers today.. and I found that I'm not able to connect to MySQL (using Sequel Pro) from my Local machine (127.0.0.1).

I couldn't discover this problem without rebuilding the mysql containers with `--no-cache` today. 

I tried solving it with no success, here's what I tried:
- I tried messing with the config file `my.cnf`.
- tried to revert all the changes made in that container and rebuild it.
- tried using the exact same old build version that is working on my machine `mysql:5.7.12`.

Nothing worked!!
